### PR TITLE
Added Drupal to rancher templates.

### PR DESCRIPTION
**Pull request by joshuacox**
This is a pretty basic drupal test environment with postgres

### CODE REVIEW

File: templates/drupal/0/rancher-compose.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      label: "DB User"
+      required: true
+      default: "drupal"
+      type: "string"
+    - variable: "DB_PASS"
+      description: "Database Password to use for the server"
+      label: "DB Password"
+      required: true
+      default: ""
+      type: "password"
```
Comment (rawmind0) [inline templates/drupal/0/rancher-compose.yml]: Only difference with version 0.1.4 (string)....
Could you please merge versions into one??

File: templates/drupal/1/rancher-compose.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      label: "DB User"
+      required: true
+      default: "drupal"
+      type: "string"
+    - variable: "DB_PASS"
+      description: "Database Password to use for the server"
+      label: "DB Password"
+      required: true
+      default: ""
+      type: "string"
```
Comment (rawmind0) [inline templates/drupal/1/rancher-compose.yml]: Only difference with version 0.1.3 (password)....
Could you please merge versions into one??

**Comment by rawmind0** [comment]
Hi @joshuacox ... Thanks so much for the PR...

I see that there is only one difference in your 2 versions. Please, take a look at change request. Could you merge your 2 versions into one?? IMHO, type password makes more sense.. :)

btw, could you please rebase and solve the files conflict?? Thanks....

**Comment by joshuacox** [comment]
looks like another users drupal is the conflict, mine uses postgres, and I'd be happy to add that in as well to get people the option.  new PR [here](https://github.com/rancher/community-catalog/pull/550)  closing this one

**Pull request closed without merging**